DUI3-DX/Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/ArcGISProjectUtils.cs

```diff
@@ -0,0 +1,66 @@
+using ArcGIS.Core.Data;
+using ArcGIS.Core.Data.DDL;
+using ArcGIS.Desktop.Core;
+using ArcGIS.Desktop.Framework.Threading.Tasks;
+
+namespace Speckle.Converters.ArcGIS3.Utils;
+
+public class ArcGISProjectUtils
+{
+  public string GetDatabasePath()
+  {
+    string fGdbName = "Speckle.gdb";
+    string fGdbPath;
+    try
+    {
+      var parentDirectory = Directory.GetParent(Project.Current.URI);
+      if (parentDirectory == null)
+      {
+        throw new ArgumentException($"Project directory {Project.Current.URI} not found");
+      }
+      fGdbPath = parentDirectory.ToString();
+    }
+    catch (Exception)
+    {
+      throw;
+    }
+    return $"{fGdbPath}\\{fGdbName}";
+  }
+
+  public string AddDatabaseToProject(string databasePath)
+  {
+    // Create a FileGeodatabaseConnectionPath with the name of the file geodatabase you wish to create
+    FileGeodatabaseConnectionPath fileGeodatabaseConnectionPath = new(new Uri(databasePath));
+    // Create actual database in the specified Path unless already exists
+    try
+    {
+      Geodatabase geodatabase = SchemaBuilder.CreateGeodatabase(fileGeodatabaseConnectionPath);
+      geodatabase.Dispose();
+    }
+    catch (ArcGIS.Core.Data.Exceptions.GeodatabaseWorkspaceException)
+    {
+      // already exists, do nothing
+    }
+
+    // Add a folder connection to a project
+    var parentFolder = Directory.GetParent(databasePath);
+    if (parentFolder == null)
+    {
+      throw new ArgumentException($"Invalid path: {databasePath}");
+    }
+    string parentFolderPath = parentFolder.ToString();
```

Comment:
can we get a new line between end of if and string for a bit of extra readability